src/pages/forms/createAccount.js

```diff
@@ -0,0 +1,312 @@
+import React, { useEffect, useRef, useState } from "react";
+import styles from "./createAccount.module.css";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router-dom";
+const CreateAccount = () => {
+  const [showEye, setShowEye] = useState(true);
+  const [showPass, setShowPass] = useState(true);
+  const [radioMarked, setRadioMarked] = useState(false);
+  const [randomVal1, setRandomVal1] = useState(true);
```

Comment:
why you still using irrelavant naming like random1 and random2 ?

for state showEye, showPass use only one state and show password based on true and false . You dont need two state values to handle one element.